src/Tokenizer/Token.php

```diff
@@ -402,14 +409,14 @@ public function isObjectOperator(): bool
     /**
      * Check if token is one of given kind.
      *
-     * @param int|list<int> $possibleKind kind or array of kinds
+     * @param _PhpTokenKind|list<_PhpTokenKind> $possibleKind kind or array of kinds
      *
      * @phpstan-assert-if-true !=null $this->getId()
      * @phpstan-assert-if-true !='' $this->getContent()
      */
     public function isGivenKind($possibleKind): bool
```

Comment:
This change is BC break.
leave it as-is in original form, trigger deprecation and introduce new `isKind(_PhpTokenKind $kind)` instead